packages/common/src/image_loaders/imgix_loader.ts

```diff
@@ -0,0 +1,62 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {ɵRuntimeError as RuntimeError} from '@angular/core';
+
+import {RuntimeErrorCode} from '../errors';
+
+import {ImageLoader, ImageLoaderConfig} from './image_loader';
+
+/**
+ * Function that generates a built-in ImageLoader for Imgix.
+ *
+ * @param path path to the desired Imgix origin
```

Comment:
nit: I think it'd be great to include an example here, so that it's more obvious what the path format is.
```suggestion
 * @param path path to the desired Imgix origin, e.g. https://somepath.imgix.net/
```